clients/drcachesim/drpt2trace/drpt2trace.cpp

```diff
@@ -74,9 +74,18 @@ static droption_t<std::string>
               "Specifies the file path of the PT raw trace. Please run the "
               "libipt/script/perf-read-aux.bash script to get PT raw trace file from the "
               "data generated by the perf record command.");
+static droption_t<std::string>
+    op_elf(DROPTION_SCOPE_FRONTEND, "elf", "", "[Optional] Path of the elf file",
+           "Specifies the file path of the elf file. You must specify when "
+           "converting short traces that don't contain sideband information.");
+static droption_t<unsigned long long>
+    op_elf_base(DROPTION_SCOPE_FRONTEND, "elf_base", 0x0,
+                "[Optional] The base address of the elf file",
+                "Specifies the base address of the elf file.");
+
 static droption_t<std::string>
     op_primary_sb(DROPTION_SCOPE_FRONTEND, "primary_sb", "",
-                  "[Required] Path of primary sideband stream file",
+                  "[Optional] Path of primary sideband stream file",
                   "Specifies the file path of the primary sideband stream. A primary "
```

Comment:
Clarify in the these docs that you either need a sideband file or if there isn't one (which should only happen when just one binary was involved, right?) you have to pass that in as --elf.

Reply:
Done.